src/llmcompressor/transformers/finetune/README.md

```diff
@@ -116,10 +115,8 @@ oneshot(
     model=model,
     dataset=dataset_name,
     concatenate_data=concatenate_data,
-    output_dir=output_dir,
```

Comment:
Considering the example is using a string, we should keep the output dir since we're not showing the saving step in the readme